app/javascripts/main/extServer.js

```diff
@@ -40,21 +41,16 @@ async function handleRequest(req, res) {
 
 function onRequestError(error, res) {
   console.error(error);
-  const FILE_DOES_NOT_EXIST = "ENOENT";
   const responseCode = error.code !== FILE_DOES_NOT_EXIST ? 404 : 500;
   res.writeHead(responseCode);
   res.write(UnableToLoadExtension);
   res.end();
 }
 
 export function createExtensionsServer() {
-  const port = 45653;
-  const ip = "127.0.0.1";
-  const host = `http://localhost:${port}/`;
-
-  http.createServer(handleRequest).listen(port, ip, () => {
+  const host = "http://localhost:45653/";
+  http.createServer(handleRequest).listen(45653, "localhost", () => {
```

Comment:
Hmm, we're duplicating constants here. 45653 is specified twice as a sort of magic number. This is why it was previously declared as a const. We can probably move these consts to the top of the file as uppercase consts.

Also see here regarding why we were binding to 127.0.0.1. I believe localhost should be the same, but be sure to double check: https://github.com/standardnotes/desktop/issues/453